CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.6.3]
+### Fixed
+- Fixed a problem related with passing the wrong method name to TransactionController ([#22102](https://github.com/MetaMask/metamask-extension/pull/22102))
```

Comment:
What was the impact on the user? Ideally this would describe changes as they affect users. It's unclear from this whether this caused a crash (and when it might be crashing), or whether it caused some operation to fail, etc.

Reply:
Thank you! And understood, I'll be a bit more descriptive next time around 💪🏼